shuup/front/api/products.py

```diff
@@ -125,7 +131,10 @@ def get_sales_unit(self, shop_product):
 
     def get_is_orderable(self, shop_product):
         customer = getattr(self.context["request"], "contact", AnonymousContact())
```

Comment:
Where is `request.contact` defined? We should probably use `get_person_contact(request.user)` here.